docs/imaging_model/README.md

```diff
@@ -0,0 +1,47 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
+
+![Imaging Model](ImagingModelTables.png)
+
+### ACLs
+
+These file types are typically not submitted to dbGaP and so can not be
+directly controlled through accession numbers and consents.
+Instead, these files will have ACLs consisting of the study kf_id and the
+most restricted set of consent codes for that participant's biospecimen.
```

Comment:
> most restricted set of consent codes for that participant's biospecimen.

"...either the consent code from the attached biospecimen (if attached directly to a biospecimen) or the most restrictive of all of the consent codes for that participant's biospecimens (if attached to a participant)."

Reply:
There's no real hierarchy to the consent codes - either people have access based on their research project or they don't. While these files might not be managed by dbGaP, we'll need to be able to take in some way whether the files are open access, or which specific consent group they should be covered under.